packages/common/src/core/schemas/hubproject.ts

```diff
@@ -185,5 +197,14 @@ export const HubProjectEditUiSchema: IUiSchema = {
         },
       ],
     },
+    {
```

Comment:
So I actually think you'll want this to be its own section on the `/edit` form. So I think you'll want something similar to the status section above, i.e: 

```ts
{
  type: "Section",
  labelKey: "{{i18nScope}}.timeline.label",
  elements: [
    {
      scope: "/properties/timeline",
      type: "Control",
      options: {
        control: "arcgis-hub-timeline-editor",
        changeEvent: "arcgisHubTimelineEditorChange",
      }
    }
  ],
}
```